appveyor.yml

```diff
@@ -1,21 +1,35 @@
+platform: x86
+
 version: '{branch}-{build}'
 
 branches:
   only:
     - master
-    - /^v\d+\.\d+$/
+    - /^v\d+\.\d+\.\d+[\w\-]*$/
 
 install:
   - "SET PATH=C:\\Python27;C:\\Python27\\Scripts;%PATH%"
   - "SET TOX_TESTENV_PASSENV=HOME USERPROFILE HOMEPATH HOMEDRIVE"
   - "pip install -U tox twine wheel"
 
-# Build the binary after tests
+# No build stage, binaries are built in 'freeze' test environment
 build: false
 
 test_script:
   - tox -e py27,freeze
 
 artifacts:
-  - path: .\dist\shub.exe
-    name: "shub Windows binary"
+  - path: dist\shub.exe
+  - path: dist\shub-$(APPVEYOR_REPO_TAG_NAME)-windows-x86.zip
+
+after_test:
+  - 7z a dist\shub-%APPVEYOR_REPO_TAG_NAME%-windows-x86.zip %APPVEYOR_BUILD_FOLDER%\dist\shub.exe
+
+deploy:
+  provider: GitHub
+  auth_token:
+    secure: mNA38grRkwGjw4zSUX1qA67eE6CRPjk97hRngZf19FuopBwbpPjOALBIlYZWKlDv
+  artifact: dist\shub-$(APPVEYOR_REPO_TAG_NAME)-windows-x86.zip
```

Comment:
It is possible to build the binary for multiples platforms, just in case, can we use `${PLATFORM}` envvar here?